localisation/wc_culture_buildings.csv

```diff
@@ -1,809 +1,22 @@
-wc_dala_macademy2;Invoker Hold;;;;;;;;;;;;;x
-wc_dala_macademy3;Invoker Covenant;;;;;;;;;;;;;x
-wc_dala_macademy4;Invoker Citadel;;;;;;;;;;;;;x
-wc_dala_witcholoks1;Witcholok Cult;;;;;;;;;;;;;x
-wc_dala_witcholoks2;Witcholok Guild;;;;;;;;;;;;;x
-wc_dala_witcholoks3;Witcholok Mission;;;;;;;;;;;;;x
-wc_dala_witcholoks4;Witcholok Tower;;;;;;;;;;;;;x
-wc_dala_tribemage1;Arcane Grove;;;;;;;;;;;;;x
-wc_dala_tribemage2;Arcane Alchemist Hut;;;;;;;;;;;;;x
-wc_dala_tribemage3;Arcanist Village;;;;;;;;;;;;;x
-wc_dala_tribemage4;Magical Enclave;;;;;;;;;;;;;x
-wc_dala_privatemage1;Arcane Holding Cells;;;;;;;;;;;;;x
-wc_dala_privatemage2;Private Arcanium;;;;;;;;;;;;;x
-wc_dala_privatemage3;Hedge Archmage Study;;;;;;;;;;;;;x
-wc_dala_privatemage4;Private Magic Academy;;;;;;;;;;;;;x
-wc_dala_muster1_desc;Dalaranian Sorcerers train in magic from a young age, becoming masters in arcane arts. Dalaran mages live in large towers which they and their apprentices to practice magic.;;;;;;;;;;;;;x
-wc_dala_muster2_desc;Dalaranian Sorcerers train in magic from a young age, becoming masters in arcane arts. A magical library, and a pain for apprentices as they try to catch flying books.;;;;;;;;;;;;;x
-wc_dala_muster3_desc;Dalaranian Sorcerers train in magic from a young age, becoming masters in arcane arts. A domed citadel with a violet roof, a common place for sorcerers to dwell in and practice or study magic.;;;;;;;;;;;;;x
-wc_dala_muster4_desc;Dalaranian Sorcerers train in magic from a young age, becoming masters in arcane arts. A magical dimension which allows safety and room to practice magic within a strange and manipulated universe, built by a cabal of powerful mages.;;;;;;;;;;;;;x
-wc_dala_armory1_desc;A mage and smith who lives in a small blacksmith with their magic tools to help forge new enchanted equipment such as staves and wands.;;;;;;;;;;;;;x
-wc_dala_armory1_desc;A magical generator used for numberous purposes, such as creating a flying town, burning away the undead, or even creating a powerful barrier.;;;;;;;;;;;;;x
-wc_dala_armory1_desc;Dalaran guilds are famed for their specialization in magic and their continued meddling in many political affairs, guild halls act as recruitment grounds and as private quarters to powerful mages.;;;;;;;;;;;;;x
-wc_dala_armory1_desc;Dalaranians collect magical artifacts of various strengths for both study and for possible use, with few limitations or restrictions. These relicquaries keep these relics under magical lock and key.;;;;;;;;;;;;;x
-wc_dala_citygarrison1_desc;Guardians are gifted heavily enchanted weapons, armor, and gear and guided by battle mage captains in Dalaranian service. Guardian barracks are violet roofed structures with a variety of magical armories to keep Guardians well supplied.;;;;;;;;;;;;;x
-wc_dala_citygarrison2_desc;Guardians are gifted heavily enchanted weapons, armor, and gear and guided by battle mage captains in Dalaranian service. The Guardian Arcanium is a magical training ground where mages
```

Comment:
Different names instead of name + number will bring only pain for players who try to do math

<details>
<summary>This is ok in mod</summary>

![Безымянный2](https://user-images.githubusercontent.com/46576860/65932483-8f885200-e416-11e9-9154-9aeeb350beb2.png)

</details>

<details>
<summary>This is broken, but you got the idea</summary>

![Безымянный3](https://user-images.githubusercontent.com/46576860/65932474-81d2cc80-e416-11e9-9f86-eb4d4ddcc33d.png)

</details>

<details>
<summary>This is vanilla family palace and this is ugly</summary>

![Безымянный5](https://user-images.githubusercontent.com/46576860/65932719-79c75c80-e417-11e9-88af-0448836d1d26.png)

</details>

<details>
<summary>This is vanilla cultural buildings and this is ok</summary>

![Безымянный4](https://user-images.githubusercontent.com/46576860/65932668-4ab0eb00-e417-11e9-9727-76268c3acdc4.png)

</details>

Reply:
I'm not sure I understand the problem or really how it pains even the poorest mathematician. Broken dryad noted, tho

Reply:
Regardless, I can make the change, it's not a huge concern to me remotely

Reply:
> I'm not sure I understand the problem or really how it pains even the poorest mathematician

Try to understand what building chain give you most troops without constantly checking the entire buildings list for building names

<details>
<summary>Click to expand</summary>

![Безымянный2](https://user-images.githubusercontent.com/46576860/65966735-34cc1600-e469-11e9-8eb4-24db85e38ab8.png)

</details>